terraform-modules/aws/vpc/variables.tf

```
@@ -74,3 +74,35 @@ variable "external_nat_ip_ids" {
   type        = list(string)
   default     = []
 }
+
+#Default Security Group Management (Default: secure)
+variable "manage_default_security_group" {
+  description = "Should be true to adopt and manage default security group"
+  type        = bool
+  default     = true
+}
+
+variable "default_security_group_name" {
+  description = "Name to be used on the default security group	"
+  type        = string
+  default     = "default"
+}
+
+variable "default_security_group_egress" {
+  description = "List of maps of egress rules to set on the default security group"
+  type        = list(map(string))
+  default     = []
+}
+
+variable "default_security_group_ingress" {
+  description = "List of maps of ingress rules to set on the default security group	"
+  type        = list(map(string))
+  default     = []
```

```suggestion
  default     = ["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16", "100.64.0.0/10"]
```

Make the default all of the RFC1918 address space and rfc6598 private address space.